examples/generate_all_certs

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
The file is not executable.

Reply:
I think I fixed this.  JFYI, my normal 'working' system is Windows (I started on DOS), and I forgot about the exe bit issue.